modules/getting-started-web-console-creating-new-project.adoc

```diff
@@ -1,33 +1,25 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-creating-new-project_{context}"]
-= Creating a new project
+= Creating a project
 
-A project enables a community of users to organize and manage their content in isolation. Projects are {product-title} extensions to Kubernetes namespaces. Projects have additional features that enable user self-provisioning.
+A _project_ enables a community of users to organize and manage their content in isolation. Projects are {product-title} extensions to Kubernetes namespaces. Projects have additional features that enable user self-provisioning. Each project has its own set of objects, policies, constraints, and service accounts.
 
-Users must receive access to projects from administrators. Cluster administrators can allow developers to create their own projects. In most cases, users automatically have access to their own projects.
+Cluster administrators can allow developers to create their own projects. In most cases, you automatically have access to your own projects. Administrators can grant access to other projects as needed.
 
-Each project has its own set of objects, policies, constraints, and service accounts.
+This procedure creates a new project called `user-getting-started`.  You will use this project throughout the rest of this tutorial.
 
 .Prerequisites
 
-* You are logged in to the {product-title} web console.
-* You are in the *Developer* perspective.
```

Comment:
@bergerhoffer this specific step requires the _Administrator_ perspective, while the next steps would require the _Developer_ perspective, so I'd get these pre-reqs back only changing the `Developer` perspective to the `Administrator` perspective.

Reply:
Ignoring per our slack convo, since 4.19 got rid of these perspectives